diff --git a/ui/build/paths/config.go b/ui/build/paths/config.go
index 78b231e9..ea6f56f1 100644
--- a/ui/build/paths/config.go
+++ b/ui/build/paths/config.go
@@ -116,8 +116,8 @@ var Configuration = map[string]PathConfig{
 	// Host toolchain is removed. In-tree toolchain should be used instead.
 	// GCC also can't find cc1 with this implementation.
 	"ar":                          Forbidden,
-	"arm-linux-androidkernel-as":  Forbidden,
-	"arm-linux-androidkernel-ld":  Forbidden,
+	"arm-linux-androidkernel-as":  Allowed,
+	"arm-linux-androidkernel-ld":  Allowed,
 	"as":                          Forbidden,
 	"cc":                          Forbidden,
 	"clang":                       Forbidden,
